examples/snippets/read-only-transactions/db/schema.rb

```diff
@@ -12,12 +12,12 @@
 
 ActiveRecord::Schema.define(version: 1) do
 
-  create_table "albums", force: :cascade do |t|
+  create_table "albums", id: { limit: 8 }, force: :cascade do |t|
```

Comment:
what is this for?

Reply:
This file is automatically generated by ActiveRecord whenever a migration is executed, and as I tested the example it got re-generated. I think that the reason that it changed this file during this execution is that the version of ActiveRecord has been updated, and it apparently now includes an explicit declaration of the length of the identifier. (The `limit: 8` in this case means that the primary key is 8 bytes, i.e. 64 bits)